tests/baselines/reference/jsxEsprimaFbTestSuite.js

```diff
@@ -72,8 +72,8 @@ baz
 <div>@test content</div>;
 <div><br />7x invalid-js-identifier</div>;
 <LeftRight left={<a />} right={<b>monkeys /> gorillas</b> /  > }/>
-    < a.b > ;
-a.b > ;
+    ,

```

Comment:
This is consistent with what we've done for a while. For example if you write
```ts
var x = a b;
```
we emit
```ts
var x = a, b;
```

As I've said before, I don't think emitting in the presence of syntax errors is a rational thing to do, but since we're doing it anyway, I think the code we emit should at least be syntactically valid (even if it makes guesses at the intent).